page_iterator_test.go

```diff
@@ -89,7 +89,7 @@ func TestIterateEnumeratesAllPages(t *testing.T) {
 
 	graphResponse := buildGraphResponse()
 	mockPath := testServer.URL + "/next-page"
-	graphResponse.SetNextLink(&mockPath)
```

Comment:
Since this is a setter, it should match the property it is setting, consider updating the`nextLink` property to `odataNextLink`